pkg/sql/conn_executor.go

```diff
@@ -1153,7 +1153,7 @@ func (ex *connExecutor) close(ctx context.Context, closeType closeType) {
 			ctx, &ex.extraTxnState.prepStmtsNamespaceMemAcc,
 		)
 		ex.extraTxnState.prepStmtsNamespaceMemAcc.Close(ctx)
-		ex.extraTxnState.sqlCursors.closeAll()
+		_ = ex.extraTxnState.sqlCursors.closeAll(false /* errorOnWithHold */)
```

Comment:
should we log any errors here just in case? same with below.

Reply:
done